sdk/src/wallet/migration/chrysalis.rs

```diff
@@ -0,0 +1,311 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use core::str::FromStr;
```

Comment:
import this from std

Reply:
Why? We import from core in many places, also in the other migrations

Reply:
It makes sense where we are in a no_std context, but that's not the case at all for the wallet so it's useless and noisy

Reply:
Done